### PR TITLE
Different colors for COVID screener

### PR DESCRIPTION
**Pull request by batemapf**
## Description
Adds in MVP functionality for different colored pass screen backgrounds for use by the Bronx VAMC in their patient screening process. The four preselected colors for the Bronx VAMC screening process are green, purple, yellow, and red. Patients are categorized into one of these colors based on COVID risk (green needs no additional screening, red requires a nasal swab and test, purple/yellow in between). Patients receive their colorized version of the screener via SMS.

Bronx VAMC is testing out this whole tiered screening flow and has agreed to roll back these changes if the flow does not reduce screening times / burden. Usage will be tracked in GA.

## Testing done
Slightly increased coverage to ensure screener loads when color is specified. Needs E2E tests to validate that color is actually displaying when specified. Manually tested.

### With no color specified
Screen Shot 2021-02-24 at 11 10 06 AM

### With color specified
Screen Shot 2021-02-24 at 12 20 30 PM

### With color and site specified
Screen Shot 2021-02-24 at 12 37 41 PM

### With color and site and language specified
Screen Shot 2021-02-24 at 12 38 33 PM

### With unrecognized color specified
Screen Shot 2021-02-24 at 11 28 59 AM

## Acceptance criteria
- [x] Color of pass screen matches color specified in URL if color specified is green, purple, yellow, or red.

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by stephenBarrs** [comment]
if the Spanish language link is clicked on a color specific page, the Spanish language page load loses the color param.  

e.g.:   

- Navigate to `http://localhost:3001/covid19screen/red`
- click on link `Cambiar al español`
-      the standard Spanish language URL loads (`http://localhost:3001/covid19screen/es`) 
-      instead of  `http://localhost:3001/covid19screen/es/red`

File: src/applications/coronavirus-screener/containers/App.jsx [hunk 1]
```diff
@@ -31,6 +32,25 @@ export default function App({ params }) {
     alternateLangauge = 'en';
   }
 
+  // If there is a color indicated, store that for the pass results screen. Customization for Bronx VAMC.
+  let passFormResultsColors = {
```
Comment (stephenBarrs) [inline src/applications/coronavirus-screener/containers/App.jsx:36]: This chunk of code can be simplified (or complicated, depending on how you look at it 😆 ) like:
```
const defaultColorSpec = {
    background: '#112e51',
    font: 'white',
    name: '',
  };
  const colorKeys = Object.keys(passFormResultsColorOptions);

  const color = colorKeys.filter(
    clr => (location.href.toLowerCase().includes(clr) ? clr : null),
  );
  const passFormResultsColors = color?.length
    ? passFormResultsColorOptions[color]
    : defaultColorSpec;
```

**Review by stephenBarrs — APPROVED** [review]
LGTM!

File: src/applications/coronavirus-screener/containers/App.jsx (hunk 1 above)
Comment (batemapf) [reply src/applications/coronavirus-screener/containers/App.jsx:36]: ah i knew there was some fancy JS shorthand to do this :-)

**Pull request merged**